dashboard/src/components/api/che-websocket.factory.js

```diff
@@ -145,15 +156,45 @@ class MessageBuilder {
 
 class MessageBus { // jshint ignore:line
 
-  constructor(datastream) {
+  constructor(datastream, $interval) {
```

Comment:
I think it is better to explain what interval means. You can rename it to pingInterval